Stanford360/Activity/Model/HealthKitManager.swift

```
@@ -112,7 +116,10 @@ class HealthKitManager {
                 quantitySamplePredicate: predicate,
                 options: .cumulativeSum
             ) { _, result, error in
-                if let error = error {
+                if let error = error as? HKError, error.errorCode == HKError.Code.errorNoData.rawValue {
+                    continuation.resume(returning: 0)
+                    return
+                } else if let error = error {
```

This is key to fixing another error. Instead of resolving the continuation by throwing an error we want to make sure that for cases when the error is due to no data, we just return 0

---

done